src/app/layout.tsx

```diff
@@ -66,6 +68,7 @@ export default function RootLayout({
             </main>
             <Toaster />
           </ThemeProvider>
+          <ReactQueryDevtools initialIsOpen={false} />
```

Comment:
Should we only install it on the Dev env?

If yes then, adding a guard here:

```suggestion
         {process.env.NODE_ENV === 'development' ? (
          <ReactQueryDevtools initialIsOpen={false} />
          ) 
          : null}
```

Reply:
It’s already automatically done by the library

https://tanstack.com/query/v4/docs/react/devtools#devtools-in-production